### PR TITLE
Remove OTA E2E and OTA PAL test for LTS 202406 release

### PR DESCRIPTION
**Pull request by chinglee-iot**
**Description of changes:**

* Remove OTA_PAL and OTA_E2E test for new MQTT file streams library for LTS 202406 release

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kar-rahul-aws — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -71,6 +66,8 @@ If testing using the [FreeRTOS-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS) re
 | [202012.04-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.04-LTS) | [202205.01](https://github.com/FreeRTOS/FreeRTOS-Libraries-Integration-Tests/tree/202205.01) |
 | [202012.05-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.05-LTS) | [202205.01](https://github.com/FreeRTOS/FreeRTOS-Libraries-Integration-Tests/tree/202205.01) |
 | [202210.00-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202210.00-LTS) | [202210.00](https://github.com/FreeRTOS/FreeRTOS-Libraries-Integration-Tests/tree/202210.00) |
+| [202210.01-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202210.01-LTS) | [202210.00](https://github.com/FreeRTOS/FreeRTOS-Libraries-Integration-Tests/tree/202210.00) |
```
Comment (kar-rahul-aws) [inline README.md:69]: The integration tests has to be (https://github.com/FreeRTOS/FreeRTOS-Libraries-Integration-Tests/tree/202210.01)

Comment (kar-rahul-aws) [inline README.md:69]: `[202210.00]` change to `[202210.01]`

Comment (chinglee-iot) [reply README.md]: Updated in new commit.

Comment (chinglee-iot) [reply README.md]: Updated in new commit.

**Review by ActoryOu — APPROVED** [review]

**Review by aggarg — APPROVED** [review]

**Pull request merged**